config/action.d/shorewall-ipset-proto6.conf

```diff
@@ -0,0 +1,84 @@
+# Fail2Ban configuration file
+#
+# Author: Eduardo Diaz
+#
+# This is for ipset protocol 6 (and hopefully later) (ipset v6.14).
+# for shorewall
+#
+# Use this setting in jail.conf to modify the name and the max time in every jain
+# action   = shorewall-ipset-proto6[name=SSH, bantime=10000]
+#
+# This requires the program ipset which is normally in package called ipset.
+#
+# IPset was a feature introduced in the linux kernel 2.6.39 and 3.0.0 kernels.
+#
+# The default Shorewall configuration is with "BLACKLISTNEWONLY=Yes" (see
+# file /etc/shorewall/shorewall.conf). This means that when Fail2ban adds a
+# new shorewall rule to ban an IP address, that rule will affect only new
+# connections. So if the attempter goes on trying using the same connection
+# he could even log in. In order to get the same behavior of the iptable
+# action (so that the ban is immediate) the /etc/shorewall/shorewall.conf
+# file should me modified with "BLACKLISTNEWONLY=No".
+#
+# A lot of this depends of the shorewall version at least have the 4.5.5 Version
+#
+# Enable to shorewall use a blacklist using iptables creating a file /etc/shorewall/blrules
+# and adding "DROP     net:+f2b-ssh    all" one for every jail
+# details in shorewall documentation blacklist.
+# to enable restore you ipset You must set SAVE_IPSETS=Yes in shorewall.conf
+# Is importan to read this documentation from shorewall http://shorewall.net/ipsets.html
+#
+# To force create the ipset in the case that somebody delete the ipset create a file
```

Comment:
I don't know some time the administrator delete the ipset if the ipset is deleted shorewall don't start, I understand that is a problem of shorewall but anyway